xhyve/xhyve.go

```diff
@@ -612,42 +612,44 @@ func (d *Driver) publicSSHKeyPath() string {
 	return d.GetSSHKeyPath() + ".pub"
 }
 
-func readLine(path string) string {
-	line := ""
+func readLine(path string) (string, error) {
 	inFile, err := os.Open(path)
 	if err != nil {
-		log.Debugf("Not able to open %s", path)
+		return "", err
 	}
 	defer inFile.Close()
+
 	scanner := bufio.NewScanner(inFile)
-	scanner.Split(bufio.ScanLines)
```

Comment:
@praveenkumar Sorry just to confirm, Is need it?
I don't understand all `bufio` package behavior. If there it, will take a more performance?